runtime/interpreter/value.go

```diff
@@ -16577,46 +16580,13 @@ func (v *CompositeValue) Destroy(interpreter *Interpreter, locationRange Locatio
 		storageID,
 		locationRange,
 		func() {
-			// if this type has attachments, destroy all of them before invoking the destructor
-			v.forEachAttachment(interpreter, locationRange, func(attachment *CompositeValue) {
```

Comment:
Do we not need to destroy attachments anymore?

Reply:
Attachments are handled in the later `forEachField` iteration we do below to handle any nested resources, since they are no longer explicitly destroyed during the invocation of the destructor. Attachments are represented as fields, so iterating over all fields will include attachments.